templates/rhsm-subscriptions-scheduler.yml

```diff
@@ -19,6 +19,10 @@ parameters:
     value: 0 3 * * *
   - name: CAPTURE_SNAPSHOT_SCHEDULE
     value: 0 1 * * *
+  - name: OPENSHIFT_METERING_SCHEDULE
+    value: 0 * * * *
+  - name: OPENSHIFT_METERING_RANGE
+    value: '180'
```

Comment:
Why 180 here? I expected to keep the default of 60?

Reply:
That was a setting for testing.  Just pushed a fix.